packages/react/src/components/Authenticator/Authenticator.tsx

```diff
@@ -27,7 +27,7 @@ function InitMachine({ children, ...data }) {
   const hasInitialized = React.useRef(false);
 
   React.useEffect(() => {
-    if (!hasInitialized.current && route === 'idle') {
+    if (!hasInitialized.current && route === 'setup') {
```

Comment:
Probably out of scope for this PR but what do you think of using `initial` as a route name in place of `setup`?

Reply:
Yep, I don't like the name `setup` either, it's too generic and doesn't say much. Let's scope this out of the PR, this ties to #1581 as well.